cluster/region/region_test.go

```diff
@@ -0,0 +1,125 @@
+package region
+
+import (
+	"github.com/ByteStorage/FlyDB/config"
+	"github.com/ByteStorage/FlyDB/engine"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+type TestRegionStruct struct {
+	region
+}
+
+func ReturnNewDB() *engine.DB {
+	opts := config.DefaultOptions
+	dir, _ := os.MkdirTemp("", "test")
+	opts.DirPath = dir
+	db, _ := engine.NewDB(opts)
+	return db
+}
+
+// NewTestRegion creates a new instance of TestRegion.
+func NewTestRegion() Region {
+	return &TestRegionStruct{
+		region{
+			id:       1,
+			startKey: []byte("start"),
+			endKey:   []byte("end"),
+			db:       ReturnNewDB(),
+			leader:   "leader",
+			peers:    []string{"peer1", "peer2"},
+			size:     100,
+		},
+	}
+}
+
+func TestRegion_Put(t *testing.T) {
+	// Create a test region instance
+	region := NewTestRegion()
+
+	err := region.Put([]byte("key_region"), []byte("value_region"))
+	assert.NoError(t, err)
```

Comment:
Add a defer statement to clean up the temporary files generated by unit tests, which will be followed by